consensus/state.go

```diff
@@ -97,11 +97,13 @@ type RoundState struct {
 }
 
 func (rs *RoundState) RoundStateEvent() types.EventDataRoundState {
+	// copy to avoid data race
```

Comment:
we don't need it before because callbacks were executed within the same context. now we spawn a thread for each event type https://github.com/tendermint/tendermint/blob/feature/548-indexing-tags/consensus/reactor.go#L317

anyway, I think transferring mutable state between 2 threads is a bad idea. events should be immutable, IMHO.